pkg/cli/server/server.go

```diff
@@ -463,6 +463,12 @@ func run(app *cli.Context, cfg *cmds.Server, leaderControllers server.CustomCont
 			cfg.DisableAgent = true
 		}
 
+		// If the user uses the cluster-reset argument in a cluster that has a ServerURL, we must return an error
+		// to remove the server flag on the configuration or in the cli
+		if serverConfig.ControlConfig.JoinURL != "" {
+			return errors.New("Please, remove the server flag when resetting the cluster")
```

Comment:
1. This is an error, not a log message - it must not be capitalized.
2. This same error will be reported on RKE2, where we instruct users to use the config file, not CLI flags - so we should be somewhat generic in what we suggest the user do.
```suggestion
			return errors.New("cannot perform cluster-reset while server URL is set - remove server from configuration before resetting")
```